colossalai/nn/optimizer/cpu_adam.py

```diff
@@ -110,7 +105,7 @@ def step(self, closure=None):
                     assert state['exp_avg_sq'].device.type == 'cpu', "exp_avg should stay on cpu"
                     self.cpu_adam_op.adam_update(self.opt_id, state['step'], group['lr'], beta1, beta2, group['eps'],
                                                  group['weight_decay'], group['bias_correction'], p.data, p.grad.data,
-                                                 state['exp_avg'], state['exp_avg_sq'], self.loss_scale)
+                                                 state['exp_avg'], state['exp_avg_sq'], -1)
```

Comment:
loss scale why -1

Reply:
We think setting loss scale in adam is against the using habit.